compiler/fir/raw-fir/psi2fir/src/org/jetbrains/kotlin/fir/builder/RawFirBuilder.kt

```diff
@@ -817,6 +817,7 @@ class RawFirBuilder(
                     isData = classOrObject.hasModifier(DATA_KEYWORD)
                     isInline = classOrObject.hasModifier(INLINE_KEYWORD)
                     isFun = classOrObject.hasModifier(FUN_KEYWORD)
+                    isExternal = classOrObject.hasModifier(EXTERNAL_KEYWORD)
```

Comment:
Let's add some external modfier tests to `RawFirBuilderLazyBodiesTestCaseGenerated`/`RawFirBuilderLazyBodiesTestCaseGenerated` which checks

Reply:
Added. I also found an issue in downward propagation of `external` and `inline` from property to default accessors in light-tree.